Error in Browser for newer version

We are using Windows 7 x64 and version 0.10 of node.js

I was using an older version of Browserify (3.20.0) and successfully compiled our project. 

However, my teammates installed the newest version (3.24.13) and the project compiles but when we try to run it in a browser we get the following error message:

Error: Cannot find module 'c:\Users\Martin Wickham\AppData\Roaming\npm\node_modules\browserify\node_modules\insert-module-globals\node_modules\process\browser.js'

--- a/mine.js
+++ b/mine.js
@@ -63,7 +63,19 @@ function mine(js) {
     if (char === quote) {
       return $close;
     }
+    if (char === "\\") {
+      return $nameEscape;
+    }
     name += char;
+    return $name;
+  }
+
+  function $nameEscape(char) {
+    if (char === "\\") {
+      name += char;
+    } else {
+      name += JSON.parse('"\\' + char + '"');
+    }
     return $name;
   }
 

--- a/test/files/windows_relative.js
+++ b/test/files/windows_relative.js
@@ -1,0 +1,1 @@
+var a = require('./a\\b\\c');

--- a/test/windows_relative.js
+++ b/test/windows_relative.js
@@ -1,0 +1,11 @@
+var test = require('tap').test;
+var mine = require('../');
+var fs = require('fs');
+var src = fs.readFileSync(__dirname + '/files/windows_relative.js');
+
+test('windows_relative', function(t) {
+  t.plan(1);
+  t.deepEqual(mine(src), [
+    {name: './a\\b\\c', offset: 17}
+  ]);
+});

